bin/tape

```diff
@@ -39,6 +50,10 @@ opts._.forEach(function (arg) {
     }
 
     files.forEach(function (file) {
+        if (matcher.shouldIgnore(file)) {
```

Comment:
instead of this, let's `files.filter(function (file) { return !matcher.shouldIgnore(file); }).forEach`?